### PR TITLE
Wizard: Use useChrome hook to set beta feature flag

### PR DESCRIPTION
**Pull request by lucasgarfield**
Calling insights.chrome is deprecated. The commits in this PR:

(1) replace isBeta() function with custom useGetEnvironment() hook
(2) adapt App.js to useChrome() hook (stop calling insights.chrome directly)
(3) replace all calls in code to insights.chrome with calls to either useChrome() or useGetEnvironment() hooks

This PR is important, because calling the deprecated insights.chrome functions results in a massive amount of warnings flooding the console, which makes it very difficult to find real/important error messages.

Chrome API docs can be found here:
http://front-end-docs-insights.apps.ocp4.prod.psi.redhat.com/chrome/chrome-api

### CODE REVIEW

File: src/Components/CreateImageWizard/CreateImageWizard.js [hunk 1]
```diff
@@ -544,11 +544,16 @@ const CreateImageWizard = () => {
...
 
   const awsTarget = isBeta() ? awsTargetBeta : awsTargetStable;
   const msAzureTarget = isBeta() ? msAzureTargetBeta : msAzureTargetStable;
-  let initialState = requestToState(composeRequest, distroInfo, isBeta());
+  let initialState = requestToState(
+    composeRequest,
+    distroInfo,
+    isBeta(),
+    isProd()
+  );
```
Comment (lucasgarfield) [inline src/Components/CreateImageWizard/CreateImageWizard.js:556]: Not very happy with how many arguments this function takes. Might need to do some refactoring in the future. Open to suggestions if anything seems obvious to you, dear reviewer.

Comment (jkozol) [reply src/Components/CreateImageWizard/CreateImageWizard.js:556]: Can we put the is beta check into our global store?

**Review by jkozol — APPROVED** [review]
Works for me and seems to reduce the error messages.

**Pull request merged**